src/lazy-script.js

```diff
@@ -0,0 +1,24 @@
+const EMPTY_GIF = 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7'
```

Comment:
This belongs to runtime.

Reply:
Located outside purely for packaging reasons. (Both `mkdist` and `siroc` clear their target directories before build.)